integration-tests/smoke/ccip_rmn_test.go

```diff
@@ -26,10 +27,8 @@ import (
 	"github.com/smartcontractkit/chainlink/v2/core/logger"
 )
 
-// Set false to run the RMN tests
-const skipRmnTest = true
-
 func TestRMN_TwoMessagesOnTwoLanes(t *testing.T) {
+	t.Parallel()
```

Comment:
Do we need `t.Parallel()` if every test is run as a separate CI job?

Reply:
Tried parallel at first did not work. can remove it